### DISCUSSION TITLE
谈谈秦始皇为啥要统一文字和思想。谈一谈啥是意识形态输出。谈一谈啥是分裂，分裂的后果是什么。

### DISCUSSION THREAD

**Discussion by islibaodong** [The-Run-Philosophy-Organization/run#642] [General]
觉得被墙了，限制了你的自由，但是这道墙他也保护着你。好好思考下，选择性的报道社会不好的一面，好的一面也没见他们歌颂啊。

每个人对世界的认识最终都是对他自己的认识。

**Comment by Pitt3** [comment]
统一文字多好啊 听不懂可以写下来啊
统一思想？ 不是很理解 有独立思考能力的人会被统一思想吗？
后面的不太懂

**Reply by islibaodong** [reply]
统一思想是指，比如：仁义礼孝啊之类的，并不是说每个人的思想必须完全统一。

**Reply by islibaodong** [reply]
还比如：自私自利，无私奉献。怎么解读无私奉献，有人说这是一种很高尚的情操，也有人说这是洗脑思想。自私自利在中国属于贬义词吧，国外呢？精致的利己主义？
